node/compile-hook.js

```diff
@@ -80,6 +83,7 @@ createWrapperManager(exts, ".mjs");
 addWrapper(Mp._compile, function(func, content, filename) {
   const options = Object.assign({ filename }, compileOptions);
   options.cacheKey = () => getCacheKey(filename);
+  runtime.enable(this);
```

Comment:
I wonder if we could make Reify work in any CommonJS environment by calling `require("reify/lib/runtime").enable(module)` at the very top of compiled modules, rather than here?